.appveyor.yml

```diff
@@ -2,6 +2,8 @@ version: 1.0.{build}
 init:
 - cmd: mkdir C:\src\
 environment:
+  signing_password:
```

Comment:
Is this **yaml** right, are you sure it doesn't need a `-` in the first column:

    - signing_password:

Reply:
:+1: OK, glad *someone* understands it... :confounded: